### ISSUE TITLE
font size of messages is too large

### ISSUE THREAD

**Issue by donovaly** [OpenSlides/OpenSlides#3033]
With OS 2.1 beta 3 it is possible to display messages with formatting as I once requested in issue #2127. There is however the problem that the message text is in a huge font size and that one cannot change the font size.

I don't think that decreasing the font size is a must have option but I think that the current fixed font size is way too large:
![localhost 8000 2017-03-03 01-30-46](https://cloud.githubusercontent.com/assets/1828501/23533351/927e5a0a-ffb1-11e6-80a1-dec6157b3ad1.jpg)

**Comment by emanuelschuetze** [comment]
Messages should be used for _short_ messages. Font size are not allowed. - If you want to write more text and resize fonts you should use a topic slide.
=> won't fix

**Comment by donovaly** [comment]
> Messages should be used for short messages. Font size are not allowed.

I agree, but id don't agree to the current font size because this is how the message looked in OS 2.1beta1 and 2:
![mitteilung-essen](https://cloud.githubusercontent.com/assets/1828501/23580489/18947f5a-0103-11e7-8955-67ba03478161.jpg)
and this is how it looks in beta3:
![localhost 8000 2017-03-04 17-51-40](https://cloud.githubusercontent.com/assets/1828501/23580495/4b91225a-0103-11e7-9387-90c225141f60.jpg)

(This screenshot is used for the Tutorial.)

So please just bring back the old font size and I am happy.

**Comment by emanuelschuetze** [comment]
The font size of the message overlay has not changed. We still use `font-size: 2.75em`.
BUT: I can see in your screenshots that you use a different resolution for the projectors. The first projector has a higer resolution - that seems the reason why the message font looks smaller.

**Comment by donovaly** [comment]
> I can see in your screenshots that you use a different resolution for the projectors.

Until now I did not know that the projector resolution could be changed. So 1280 x 960 was in the past the default resolution. Now the default is 1024 x 768. This is a 4:3 resolution. As I haven't seen a beamer with this resolution, I don't think this default is very useful. if you prefer a 4:3 resolution the default should be 1280 x 960 pixel.

(FYI, I described the projector resolution feature now in the docs.)
